package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-core-sdk",
-    "version": "3.5.0",
+    "version": "3.6.0",
```

Comment:
I don't know when it became the new normal to include a bump in every PR. When we release using `taoRelease npmRelease` it should be added automatically. It's harmless, but for now I'm not sure if your published package will be 3.6.0 or 3.7.0.

Reply:
Should I roll this back?

Reply:
Anyway. [Removed](https://github.com/oat-sa/tao-core-sdk-fe/compare/f7010e0fad756976fff1a2784f90c4823d711437..ad8060f743978ee9c2feb19bf3cb9e4c93327e01) the change.